OasisDEX/oasis-borrow
# Uniswap widget

# [Uniswap widget](https://app.shortcut.com/oazo-apps/story/3618/uniswap-widget)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
 
- Add an exchange button at the top which opens the widget.
- It's disabled on mobile.
  
## How to test 🧪

- Enable the feature on your browser console: `localStorage.setItem('features', '{"Exchange":true}')`
- On goerli it just has two tokens, on mainnet you would see all the chosen tokens.
